embulk-output-redshift/src/main/java/org/embulk/output/RedshiftOutputPlugin.java

```diff
@@ -83,6 +83,14 @@
         @Config("ssl")
         @ConfigDefault("\"disable\"")
         public Ssl getSsl();
+
+        @Config("copy_unload_iam_role_name")
```

Comment:
'unload' is unnecessary because embulk-output-redshift doesn't use UNLOAD.
Namely, the option name should be 'copy_iam_role_name'.

Reply:
OK, I fix it.